docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -57,4 +57,21 @@ kubectl wait --for=condition=Available deployment/lifecycle-operator -n keptn-li
 
 The Lifecycle Toolkit and its dependencies are now installed and ready to use.
 
-</details>
+> **Note**
```

Comment:
what happened here? did you intentionally change the content?

Reply:
https://github.com/keptn/lifecycle-toolkit/pull/1031#discussion_r1156777688